### PR TITLE
Validation global error

### PR DESCRIPTION
**Pull request by balanza**
# Description

Add `getGlobalError` utility to map server validation error that not relate to a field. The use case is to render failures coming from the server when fetching/saving data.

This PR also applies the error handling as an improvement of https://github.com/trento-project/web/pull/2727.

See unit tests for expected usage.

### CODE REVIEW

File: assets/js/common/ActivityLogsSettingsModal/ActivityLogsSettingsModal.test.jsx [hunk 1]
```diff
@@ -5,6 +5,7 @@ import userEvent from '@testing-library/user-event';
 import '@testing-library/jest-dom';
 
 import ActivityLogsSettingsModal from '.';
+import { defaultGlobalError } from '../../lib/api/validationErrors';
```
Comment (nelsonkopliku) [inline assets/js/common/ActivityLogsSettingsModal/ActivityLogsSettingsModal.test.jsx:8]:
```suggestion
import { defaultGlobalError } from '@lib/api/validationErrors';
```

File: assets/js/state/sagas/activityLogsSettings.js [hunk 2]
```diff
@@ -11,6 +11,7 @@ import {
   setEditingActivityLogsSettings,
   setNetworkError,
 } from '@state/activityLogsSettings';
+import { defaultGlobalError } from '../../lib/api/validationErrors';
```
Comment (nelsonkopliku) [inline assets/js/state/sagas/activityLogsSettings.js:14]:
```suggestion
import { defaultGlobalError } from '@lib/api/validationErrors';
```

File: assets/js/state/sagas/activityLogsSettings.test.js [hunk 3]
```diff
@@ -17,6 +17,7 @@ import {
   fetchActivityLogsSettings,
   updateActivityLogsSettings,
 } from './activityLogsSettings';
+import { defaultGlobalError } from '../../lib/api/validationErrors';
```
Comment (nelsonkopliku) [inline assets/js/state/sagas/activityLogsSettings.test.js:20]:
```suggestion
import { defaultGlobalError } from '@lib/api/validationErrors';
```

**Review by nelsonkopliku — APPROVED** [review]
Super, thanks!

Just a few paths to adjust, if you want, for the rest  LGTM.

I am now realizing that in SUMA credentials modal there is no generic failure handling afaik, so this change could be beneficial there as well.
(I mean, you don't have to do that. Just saying that we now have the opportunity to improve other parts of the app :wink: )

**Comment by balanza** [comment]
@nelsonkopliku super thanks for the suggestion, I actually used the automatic import but it's not that savvy... I'll find out if there are any eslint rules we can enforce, though

**Pull request merged**